src/components/util/document-store.tsx

```diff
@@ -0,0 +1,28 @@
+import { getWalletDetails } from "./wallet";
+
+export const setDocumentStoreInformation = async (documentStoreAddress: string): Promise<void> => {
```

Comment:
naming is not very good
- storeDocumentStoreInLocalStorage

same for the other method

Reply:
okay,updated.

Reply:
is *from* better than *in* for: `retrieveDocumentStoreInLocalStorage` ?